gammapy/maps/core.py

```diff
@@ -427,7 +429,7 @@ def write(self, filename, overwrite=False, **kwargs):
         hdulist.writeto(make_path(filename), overwrite=overwrite, checksum=checksum)
 
     def iter_by_axis(self, axis_name, keepdims=False):
-        """ "Iterate over a given axis
+        """Iterate over a given axis.
 
         Yields
```

Comment:
Yields? Is this a common word for docstring?

Reply:
Don't know but it is what replace `return` in a generator so why not !